config.toml

```diff
@@ -13,6 +13,16 @@
 # directory that you are sharing to the printer through the USB Gadget
 files_directory = "/mnt/usb_share"
 
+[file_list]
+# The sort order for the file list in the frontend. Can be:
+#   * "alpha" - for Alphanumeric sort
+#   * "creation" - for creation date/time sort
+#sort_order = "creation"
```

Comment:
nit: can you call this `sort_by` instead? “order” implies in being able to set if it’s in ascending or descending order, while `sort_by` makes it clearer that it’s about the field in which we are sorting